pandas/tests/reshape/test_pivot.py

```diff
@@ -742,24 +742,27 @@ def test_margins_no_values_two_row_two_cols(self):
             index=['A', 'B'], columns=['C', 'D'], aggfunc=len, margins=True)
         assert result.All.tolist() == [3.0, 1.0, 4.0, 3.0, 11.0]
 
-    def test_pivot_table_with_margins_set_margin_name(self):
+    @pytest.mark.parametrize(
+        'margin_name', ['foo', 'one', 666, None, ['a', 'b']])
+    def test_pivot_table_with_margins_set_margin_name(self, margin_name):
```

Comment:
this test parametrized only to the extent to simplify expected error message construction.